src/modules/metadata/actions.js

```diff
@@ -81,6 +83,7 @@ export const fetchProjectsWithDatasetsAndTables = () => async (dispatch, getStat
     dispatch(beginFlow(FETCHING_PROJECTS_WITH_TABLES));
     await dispatch(fetchProjects());
     await dispatch(fetchProjectTables(getState().projects.itemsByID));
+    await dispatch(fetchExtraPropertiesSchemaTypes());
```

Comment:
this isn't part of fetching projects - this fetch be called elsewhere, upstream of projects being fetched - probably right before this in `fetchUserDependentData` using a Promise.all